### PR TITLE
[DOCs/operators]: 2024.5-ragusa updates

### PR DESCRIPTION
**Pull request by serinko**
# Description

Routine update of documentation with new releases.

### CODE REVIEW

File: documentation/operators/src/changelog.md [hunk 1]
```diff
@@ -2,6 +2,84 @@
...
+- [`nym-api`: make report/avg_uptime endpoints ignore blacklist](https://github.com/nymtech/nym/pull/4599): When querying for node specific data, it's no longer going to go through the entire list of all cached (and filtered nodes) to find it; instead it will attempt to retrieve a single unfiltered entry.
+    - Tests:
+        - Built the project and deployed it in a test environment.
+        - Manually tested API endpoints for mixnode and gateway data.
+        - Verified that the endpoints return the expected data and handle blacklists correctly.
+        - API performance improved due to the efficient `HashMap` lookups
+        - Data in mainnet will differ from test nets due to the increased amount of gateways and mixnodes in that environment
+        - Test standard uptime routes:
+```sh
+curl -X 'GET' 'https://qa-nym-api.qa.nymte.ch/api/v1/status/gateway/3ZmKvV3Fax9A8txxQ3GKh9fUeyTAk68rK4Yn4m3Vdvp9/avg_uptime' -H 'accept: application/json'
```
Comment (tommyv1987) [inline documentation/operators/src/changelog.md]: we can use the mainnet data here now for querying this if need be for an example as this has been updated in mainnet
```
curl -X 'GET' 'https://validator.nymtech.net/api/v1/status/gateway/Fo4f4SQLdoyoGkFae5TpVhRVoXCF8UiypLVGtGjujVPf/avg_uptime' -H 'accept: application/json'

Comment (serinko) [reply documentation/operators/src/changelog.md]: great! thank you

**Comment by serinko** [comment]
Ready for review @tommyv1987 / @mfahampshire / @benedettadavico

File: documentation/operators/src/changelog.md [hunk 2]
```diff
@@ -2,6 +2,160 @@
...
+[#4564]: https://github.com/nymtech/nym/pull/4564
+[#4553]: https://github.com/nymtech/nym/pull/4553
+[#4552]: https://github.com/nymtech/nym/pull/4552
+[#4548]: https://github.com/nymtech/nym/pull/4548
+~~~
+
+### Features
+
+- New `nym-node` API endpoint `/api/v1/auxiliary-details`: to expose any additional information. Currently it's just the location. `nym-api` will then query all nodes for that information and put it in the `self-described` endpoint.
+- Newly `nym-node` location available - use one of the three options to add this to your node config:
```
Comment (mfahampshire) [inline documentation/operators/src/changelog.md]: Should be `new` ?

File: documentation/operators/src/nodes/nym-node.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Once VPS and Nym wallet are configured, binaries ready, the operators of `nym-no
 
 During the testing events series [Fast and Furious](https://nymtech.net/events/fast-and-furious) we found out, that after introducing IP Packet Router and [Nym exit policy](https://nymtech.net/.wellknown/network-requester/exit-policy.txt) as default features,  only a fragment of Exit Gateways routes correctly through IPv4 and IPv6. We built a useful monitor to check out your Gateway (`nym-node --mode exit-gateway`) at [harbourmaster.nymtech.net](https://harbourmaster.nymtech.net/).
 
-Below is a fast - ten commands - deployment for seasoned operators to migrate and setup the node, configure networking and connectivity and verify that it all works as it should by getting two free jokes through the Mixnet.
+Below is a fast - *ten commands* - deployment for seasoned operators to migrate and setup the node, configure networking and connectivity and verify that it all works as it should by getting two free jokes through the Mixnet.
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/nym-node.md]: 'commands' should be 'command'

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: documentation/operators/src/changelog.md (hunk 2 above)
Comment (serinko) [reply documentation/operators/src/changelog.md]: done

File: documentation/operators/src/nodes/nym-node.md (hunk 3 above)
Comment (serinko) [reply documentation/operators/src/nodes/nym-node.md]: done

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**